contracts/ActivePool.sol

```diff
@@ -87,14 +91,12 @@ contract ActivePool is
 	}
 
 	function setStakingAdminAddress(address _stakingAdminAddress) external {
-		require(!isStakingAdminInitialized, "ActivePool: staking admin already initialized");
+		require(stakingAdmin == address(0), "ActivePool: staking admin already initialized");
```

Comment:
sorry I was too late to update my comment, but maybe something like this instead

`stakingAdmin == address(0) || stakingAdmin == msg.sender`. I really doubt it we'll use it, but in case we need to change the admin.